servers/core/coursePhaseType/initializeTypes.go

```
@@ -365,3 +365,62 @@
 
 	return nil
 }
+
+func initSelfTeamAllocation() error {
```

golangci-lint (1.64.8)

func `initSelfTeamAllocation` is unused

(unused)